packages/flutter/lib/src/cassowary/term.dart

```diff
@@ -6,11 +6,21 @@ import 'equation_member.dart';
 import 'expression.dart';
 import 'param.dart';
 
+/// Represents a single term in an expression. This term contains a single
+/// indeterminate and has degree 1.
```

Comment:
are users of this class expected to know what "contains a single indeterminate and has degree 1" means?